contents/swift_from_zero_to_hero.playground/Pages/07_03_currying.xcplaygroundpage/Contents.swift

```diff
@@ -60,7 +60,7 @@ func addCurriedVersion(_ product: String) -> (Double) -> (Int) -> String {
 let brownPantiesWaitingForPriceInformationFunction = addCurriedVersion("Brązowe Galoty")
 
 /*:
-`brownPanties...` ma w sobie zapieczoną informację o produkcie. Jest to bezpiecznie zamknięte wewnątrz bez możliwości podmiany tego produktu. W następnym kroku powiemy tej funkcji jaka jest cena za jedną sztukę produktu i w zamian otrzymamy kolejną funkcje!
+`brownPanties...` ma w sobie zabezpieczoną informację o produkcie. Jest to bezpiecznie zamknięte wewnątrz bez możliwości podmiany tego produktu. W następnym kroku powiemy tej funkcji jaka jest cena za jedną sztukę produktu i w zamian otrzymamy kolejną funkcje!
```

Comment:
Ma być zapieczoną :) Idea domknięć jest taka, że "zapamiętują" to co jest w nich użyte. Obawiam się, że komus może się to pomylić z "bezpieczeństwem" a troche inna idea jest w tym zdaniu :)

Reply:
może w takim razie zapieczętowaną albo zaplombowaną?

Reply:
zapieczetowana brzmi spoko :)